Userland/DevTools/HackStudio/HackStudioWidget.cpp

```diff
@@ -907,10 +912,45 @@ void HackStudioWidget::set_current_editor_wrapper(RefPtr<EditorWrapper> editor_w
     update_tree_view();
 }
 
+void HackStudioWidget::rename_file(String const& old_name, String const& new_name)
```

Comment:
I feel like this name is not quite right since the renaming in the file system is not actually happening here. This is more of a "change a bunch of stuff that referred to a renamed file with its old name"

Reply:
Hmm.. How about `file_renamed` to indicate that the function is supposed to be called _after_ the actual renaming?